test/unit/test_utils.js

```diff
@@ -79,6 +79,7 @@ class XRefMock {
     this._map = Object.create(null);
     this.stats = new DocStats({ send: () => {} });
     this._newRefNum = null;
+    this.stream = { isDataLoaded: true };
```

Comment:
What change(s) made this necessary?

Also, please let's avoid any future issues by providing a "valid" `BaseStream`-instance here:
```suggestion
    this.stream = new NullStream();

Reply:
In the test:
https://github.com/mozilla/pdf.js/pull/15082/files#diff-63129533f0fcab1395066c5eecb394e84cc3538b5e016a2f0bb49942825610e6R4088
we have to load the resources:
https://github.com/mozilla/pdf.js/blob/d72a85fa460ed3cf0f6914834cf2fa8b65ec8071/src/core/annotation.js#L857
and the `isDataLoaded` is used:
https://github.com/mozilla/pdf.js/blob/d72a85fa460ed3cf0f6914834cf2fa8b65ec8071/src/core/object_loader.js#L66